jenkins/aws/buildPython.sh

```diff
@@ -81,7 +81,13 @@ function main() {
       info "Running unit tests with pytest..."
       if [[ -n ${MANAGE_OPTIONS} ]]; then
         # Set --junitxml option if TEST_REPORTS_DIR and TEST_JUNIT_DIR are set
-        MANAGE_OPTIONS=" --junitxml="+"${MANAGE_OPTIONS}"
+        MANAGE_OPTIONS=" --junitxml=${MANAGE_OPTIONS} --cov"
```

Comment:
Should --cov be under the COVERAGE_REPORT statement?

Reply:
the only thing, that coverage must be installed as a requirement, but it would be obvious it is missing. I think it is good to have it running by default.